### DISCUSSION TITLE
Is v6 supports grouping(naming components and hierarchy)?

### DISCUSSION THREAD

**Discussion by bang9** [storybookjs/react-native#328] [Q&A] [answered]
When I used `storiesOf` of v5, I can create a subgroup with `add`, but I wonder if v6 can be configured in the same format.

The image below is a screenshot of a web storybook brought, for example.
In the image, grouping is configured by 3-depth, but 2-depth is enough in my case.
![image](https://user-images.githubusercontent.com/26326015/153762661-dc7f15e8-7b61-4166-82be-cdf69a9c0391.png)

**Comment by dannyhw** [comment, marked as answer]
@bang9 its not yet implemented but its something thats planned for an upcoming release.

**Reply by bang9** [reply]
I see, thanks!
